website/templates/nav.mako

```diff
@@ -13,9 +13,9 @@
             </button>
         <a class="navbar-brand" href="/" aria-label="Go home"><span class="osf-navbar-logo"></span></a>
         <div class="service-name">
-            <a data-bind="attr: {href: osfServices[currentService].href}">
+            <a href="${service_href()}">
                 <span class="hidden-xs"> OSF </span>
-                <span><strong data-bind="text: osfServices[currentService].name"></strong></span>
+                <span><strong>${service_title()}</strong></span>
```

Comment:
I think I've figured out a way we can avoid this weird function-passing. It involves changing this nav file from using a `block` to a `def`. I'll send a PR shortly.